code/controllers/subsystem/ticker.dm

```diff
@@ -129,6 +129,10 @@ SUBSYSTEM_DEF(ticker)
 
 	..()
 	start_at = world.time + (CONFIG_GET(number/lobby_countdown) * 10)
+	if(CONFIG_GET(flag/randomize_shift_time))
```

Comment:
Add another config flag that makes shift time pull from the actual realworld time in the server's timezone or I'll make sure you'll never have your tabular desires sated ever again.